Vagrantfile

```diff
@@ -270,7 +270,10 @@ Vagrant.configure(2) do |config|
     end
     cilium_dir = '.'
     cilium_path = '/home/vagrant/go/src/github.com/cilium/cilium'
-    if ENV["SHARE_PARENT"] then
+    if ENV["SHARE_PARENT"] == "2" then
+      cilium_dir = '../..'
+      cilium_path = '/home/vagrant/go/src/github.com'
+    elsif ENV["SHARE_PARENT"] then
```

Comment:
Might be worth checking for `== "1"` here, in case some contributors see the `SHARE_PARENT=2` and try to disable by explicitely passing `SHARE_PARENT=0`.